Bottom: 76e6fa286b8d4fddfe74b05e278fe368906730a3
Top:    aed932240623fe68d64cafd934bce354a91d1031
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-06 13:24:50 +0800

Refresh of autofs-5.0.5-fix-master-map-bogus-keywork-match.patch

---

diff --git a/lib/master_tok.l b/lib/master_tok.l
index 5d378c3..dfdcb14 100644
--- a/lib/master_tok.l
+++ b/lib/master_tok.l
@@ -92,7 +92,7 @@ NL		\r?\n
 CONT		\\\n{OPTWS}
 
 OPTIONSTR	([\-]?([[:alpha:]_]([[:alnum:]_\-])*(=([[:alnum:]_\-])+)*)+)
-QVOPTIONSTR	([\-]?([[:alpha:]_]([[:alnum:]_\-])*="([[:alnum:]_\-\:])+"))
+QVOPTIONSTR	([[:alpha:]_]([[:alnum:]_\-])*="([[:alnum:]_\-\:])+")
 MACROSTR	(-D{OPTWS}([[:alpha:]_]([[:alnum:]_\-\.])*)=([[:alnum:]_\-\.])+)
 SLASHIFYSTR	(--(no-)?slashify-colons)
 NUMBER		[0-9]+
